modules/apps/depot/depot-test/src/testIntegration/java/com/liferay/depot/web/internal/util/test/DepotAdminGroupSearchProviderTest.java

```diff
@@ -196,7 +197,9 @@ private ThemeDisplay _getThemeDisplay() throws Exception {
 		ThemeDisplay themeDisplay = new ThemeDisplay();
 
 		themeDisplay.setCompany(_company);
+		themeDisplay.setLocale(LocaleUtil.getDefault());
 		themeDisplay.setScopeGroupId(_group.getGroupId());
+		themeDisplay.setUser(TestPropsValues.getUser());
```

Comment:
@AliciaGarciaGarcia, it would be better to also add themeDisplay.setRealUser() here. 

There are sometimes issues when running the test on a clean bundle where the user hasn't logged in yet

Reply:
adding to the commit